CMakeLists.txt

```diff
@@ -111,7 +114,21 @@ set(NRN_RX3D_OPT_LEVEL
 # =============================================================================
 # Set Python additional versions earlier (especially for old CMake)
 # =============================================================================
-set(Python_ADDITIONAL_VERSIONS 3 3.9 3.8 3.7 3.6 3.5 3.4 3.3 3.2 3.1 3.0 2.8 2.7 2.6)
+set(Python_ADDITIONAL_VERSIONS
+    3
+    3.9
+    3.8
+    3.7
+    3.6
+    3.5
+    3.4
+    3.3
+    3.2
+    3.1
+    3.0
+    2.8
+    2.7
+    2.6)
```

Comment:
@alexsavulescu : this is quite inconvenient. I suggest to disable cmake-format for this line using `# ~~~` marker.

Reply:
Ok reverted, we can do a separate PR.